eng/pipelines/templates/steps/build-test.yml

```diff
@@ -70,7 +73,7 @@ steps:
     inputs:
       targetType: 'filePath'
       filePath: ./eng/scripts/run_tests.ps1
-      arguments: '${{ parameters.ServiceDirectory }} ${{ parameters.TestRunTime }}'
+      arguments: '${{ parameters.ServiceDirectory }} ${{ parameters.TestRunTime }} ${{ parameters.EnableRaceDetector }}'
```

Comment:
@benbp is there some magic I can do here to pass a proper PS `$true`? It's passing `True` which the PS script doesn't like.

Reply:
I tried adding a leading `$`, let's see if that works.

Reply:
It worked.